sda-commons-server-spring-data-mongo/src/main/java/org/sdase/commons/server/spring/data/mongo/MongoConfiguration.java

```diff
@@ -115,7 +117,7 @@ public MongoConfiguration setHosts(String hosts) {
   @Deprecated(forRemoval = true)
   public String getDatabase() {
     if (StringUtils.isBlank(database) && StringUtils.isNotBlank(connectionString)) {
-      return String.join(",", new ConnectionString(connectionString).getDatabase());
+      return new ConnectionString(connectionString).getDatabase();
```

Comment:
If creating the `ConnectionString` is expensive, we could also store it as a field of this class.

Reply:
That only happens on startup. Not sure if it's worth it.